.github/workflows/prbuild.yaml

```diff
@@ -24,6 +24,12 @@ jobs:
       with:
         name: image
         path: image/contour-*.tar
+    - uses: act10ns/slack@v1
+      with:
+        status: ${{ job.status }}
+        steps: ${{ toJson(steps) }}
+        channel: '#contour-ci-notifications'
+      if: ${{ failure() && github.head_ref == 'refs/heads/main' }}
```

Comment:
turns out `github.ref` is the thing to use for `main` builds (worked on operator)

Reply:
docs: https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#github-context

Reply:
👍